qiskit/providers/aer/noise/device/parameters.py

```diff
@@ -96,7 +97,8 @@ def gate_time_values(properties):
         name = gate.gate
         qubits = gate.qubits
         value = None  # default value
-        params = _check_for_item(gate.parameters, 'gate_time')
+        params = _check_for_item(gate.parameters, 'gate_time')  # deprecated
```

Comment:
This is useless, right? The line behind will always overwrite `params`

Reply:
Yeah you're right, should be a different variable name

Reply:
Since this wasn't actually used in old version of backends I'll just remove it